### ISSUE TITLE
Search Alnair Brings up Zeta Centauri

### ISSUE THREAD

**Issue by TesseractPi** [Stellarium/stellarium-web-engine#129]
If one does a search on Alnair, Zeta Centauri comes up instead (different star; different constellation).

If one goes to the constellation Gruis and selects its alpha star, Alnair and its details are correctly reflected.  So, the map knows Alnair, but the search link is off.

This was observed via the online website, via entry with an Android mobile phone.  It may be an issue regardless of how one gains access to the website, but I could only check via an Android cellphone.

![Screenshot_2021-12-20-14-04-19-1](https://user-images.githubusercontent.com/95396065/146825565-98eb1c13-c1d0-4673-b87a-0261f3eef639.png)

**Comment by gzotti** [comment]
This is not wrong, historically. _Al-Nair_ ("the bright one") is a part of the name of several stars, although it should be completed by more details. The search result should however be a short-list, where the user should be able to select from all entries. If that is hard to implement and search order depends on the order in the data file, the "official" IAU list should be advanced to the top.

**Comment by TesseractPi** [comment]
Yes it is wrong.  Alnair is the alpha member of Gruis.  It is in no manner Zeta Centauri.Search logic should be uniform for all objects.  All other searches, at least as far as I've seen, link to the star name -- not its attributes.How does one find the star Alnair then?Regardless, then searching on "Al-nair" as a way to say the customer wants to view a "bright one" would direct where?  To any number of the first 31 brightest stars (since Alnair is the 31st or 32nd brightest star)?  Zeta Centauri, with an apparent magnnitude of ~2.55 is way down on the list (it certainly isn't a "bright one").Searching on "Alnair" should work as other seatches do:  on the actual name of the star, not a conceptualization that would be unknown to any but the one who programmed it.  At present, the search does not work.On 12/20/21, 15:38 Georg Zotti ***@***.***> wrote:

   
  This is not wrong, historically. Al-Nair ("the bright one") is a part of the name of several stars, although it should be completed by more details. The search result should however be a short-list, where the user should be able to select from all entries. If that is hard to implement and search order depends on the order in the data file, the "official" IAU list should be advanced to the top.

**Comment by gzotti** [comment]
OK, you don't understand that star names have a history, and some names have been used for more than one star. You could do just a little bit of research to learn.  I have written above what we could do to improve this situation.

**Comment by TesseractPi** [comment]
Okay, if you want to be consistent and make it uniformly understandable for customers - and you personally favor using Alnair as the main [search-engine triggered] name for Zeta Centauri ...Then change the main name field from Zeta Centauri to Altair.  But doing so, I would hope one does not  change to main name of Alnair [Alpha Gruis] to Alpha Gruis.You could then have both Alnair's show up on the search down list.  Specifically, the minor Alnair [Zeta Centauri], which is the more obsecure, historically usage -- and the major Alnair [Alpha Gruis, the first magnitude star].Alnair as used for Zeta Centauri appears mostly in short-lived and now outdated ephermeris, an ephemera reference, way back whenI was a kid.  It's also referenced in Wikipedia, but that source is a depository for all sorts of valid or favored source-material nomenclatures.On the otherhand, Alnair is the historical, standard and more universally accepted name for Alpha Gruis.Whoever designed the detail fields for Zeta Centauri must have, correctly, understood that the Alnair designation for it was less substantive.  I'd venture to say that you at least partially agree with the point of my issue [that the search should have used Zeta Centauri for that more minor star, and Alnair for Aloha Gruis --  else you would not have inappropriately insulted me as an attempt to deflect and admit the isdue dies ecist and should be addressed.Unfortunately, in reviewing how you tend yo reply to other issue submitters, this seems to be your M.O.In reviewing how you respond to many individuals who submit issues, I guess there's no point in submitting issues on GitHub's Stellarium page if the designers view such as nefmfative instead of positive; if they becime defensive and insulting instead of appreciating our efforts to help them improve the software.If you are Stellarium's only volunteer coder, I guess you can, unfortunately, act as both gatekeeper and ridiculer of any issue that happens to notice a bit of your code that could be improved upon. I'm not sure if you really believe I "don't know that star names have a history" -- a false and insulting assumption.  It does little but closes the open-source environment of GitHub by responding in your habitual flippant and defensive manner (not just to me, but to the many others whose backgrounds and experience you know nothing about).We take the time to identify and log potential issues -- not because we are trying to make a defensive programmer look bad.  We aren't; we are trying to help make the software better than it already is.  For that to be seen as a negative -- or worse, make us feel as though we never wish to contribute again -- might assuage the egos of insecure designers, but it also deminishes GitHub's goal of improving wuslirt AND SALES of their clients' software.Before I became involved in contributing to GitHub, I wanted to buy the full versions of Stellarium (once a few minor issues were addressed).  Thanks to GitHub's allowing these unprofessional and insulting responses to issue submissions -- immature attitudes that do much to CLOSE the door on open-source participation -- I have comoketely lost interest in both GitHub and Stellarium. 

 
  
  
 
 On 12/20/21, 16:42 Georg Zotti ***@***.***> wrote:

   
  OK, you don't understand that star names have a history, and some names have been used for more than one star. You could do just a little bit of research to learn. I have written above what we could do to improve this situation.
